asv_bench/benchmarks/categoricals.py

```diff
@@ -43,3 +45,16 @@ def time_regular_constructor(self):
     def time_fastpath(self):
         Categorical(self.codes, self.cat_idx, fastpath=True)
 
+
+class categorical_rendering(object):
+    goal_time = 2e-3
+
+    def setup(self):
+        n = 1000
+        items = [str(i) for i in range(n)]
+        s = pd.Series(items, dtype='category')
+        df = pd.DataFrame({'C': s, 'data': np.random.randn(n)})
+        self.data = df[df.C == '20']
+
+    def time_rendering(self):
+        str(data.C)
```

Comment:
or maybe, that's meant to happen, as it tests for all to be tested environment? I don't know, have never used asv before.